test/main_test.go

```diff
@@ -121,7 +121,6 @@ func configureTestOutput() {
 		panic(fmt.Errorf("Error creating junit file: %v", err))
 	}
 
-	opt.Format = "junit"
```

Comment:
Why did you remove this line?

Reply:
I noted it in the PR description, but it's true that it may have got lost among other things:

`I have also removed unnecessary format configuration as it's already in the options.`

The thing is that first I tried to change it to pretty in options struct on the top of the file, but then found out that this setting was overriding it. So why have 2 places where it is set up when we have it already in the options struct on the top.

Reply:
oh, ok, I see.
Fine for me.